protocol/msgpack/types.go

```diff
@@ -344,3 +365,23 @@ type AggregatedIteratorPool interface {
 	// Put puts an aggregated iterator into the pool
 	Put(it AggregatedIterator)
 }
+
+// PolicyCompressionOptions provides options for policy compression
+type PolicyCompressionOptions interface {
+	// Enabled returns whether policy compression is enabled or not
+	Enabled() bool
+
+	// SetEnabled sets the enabled status of
+	SetEnabled(enabled bool) PolicyCompressionOptions
+
+	Compressor() policy.Compressor
+
+	SetCompressor(compressor policy.Compressor) PolicyCompressionOptions
+}
+
+// PolicyDecompressionOptions provides options for policy decompression
+type PolicyDecompressionOptions interface {
+	Decompressor() policy.Decompressor
```

Comment:
Perhaps add `Enabled` just like the `PolicyCompressionOptions`? Also what are your thoughts on just calling them `CompressionOptions` in case in the future we need to add id compression too?